.dir-locals.el

```diff
@@ -0,0 +1,11 @@
+;;; Directory Local Variables
+;;; For more information see (info "(emacs) Directory Variables")
+
+((c++-mode . ((indent-tabs-mode . t)
+              (tab-width . 8)
+              (c-basic-offset . 8)
+              (ethan-wspace-errors . (many-nls-eof no-nl-eof eol))))
+ (c-mode . ((indent-tabs-mode . t)
+            (tab-width . 8)
+            (c-basic-offset . 8)
+            (ethan-wspace-errors . (many-nls-eof no-nl-eof eol)))))
```

Comment:
Is there a vim equivalent of this? Might be good to have if non-emacs users are planning to edit the code, since this seems like an easy thing to mess up without realising